src/style.css

```diff
@@ -1 +1,121 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The use of the `*` selector is discouraged because it impacts performance. It is more efficient to set styles only for elements that require them.